tasks/models/workflow.py

```diff
@@ -91,13 +91,17 @@ def get_task_templates(self) -> models.QuerySet:
         )
 
     @atomic
-    def create_task_workflow(self) -> "TaskWorkflow":
-        """Create a workflow and it subtasks, using values from this template
-        workflow and its task templates."""
+    def create_task_workflow(self, summary_data: dict) -> "TaskWorkflow":
```

Comment:
Might it be preferable, better documenting this API, to be explicit stating which fields are required or expected to create a `TaskWorkflow`, with `TaskWorkflow.summary_task` being kept as an internal detail?
```suggestion
    def create_task_workflow(self, title: str, description: str) -> "TaskWorkflow":
```